autofaiss/indices/distributed.py

```diff
@@ -136,7 +140,8 @@ def _merge_index(
     end: Optional[int] = None,
     max_size_on_disk: str = "100GB",
     tmp_output_folder: Optional[str] = None,
-) -> faiss.Index:
+    index_optimizer: Callable = None,
+) -> Tuple[faiss.Index, Dict[str, str]]:
     """Merge all the indices in `small_indices_folder` into single one return the merged index."""
```

Comment:
We are not only returning the merged index now, comment should be updated.

Reply:
done